src/app/features/auth/auth.module.ts

```diff
@@ -10,11 +11,13 @@ import {
  * Lazy loaded feature module for all auth-related components and dependencies.
  */
 @NgModule({
+    providers: [TNSFontIconService],
     declarations: [
         authComponents
     ],
     imports: [
         AuthRoutingModule,
+        TNSFontIconModule,
```

Comment:
Might we need to move this to SharedModule later

Reply:
We have to have this in all modules.

Reply:
Right, But it would be nice to have it in SharedModule later.

Reply:
Agree